dask/array/core.py

```diff
@@ -929,13 +936,21 @@ def store(sources, targets, lock=True, regions=None, compute=True,
         store_dsks, tgt_dsks, [sources_dsk]
     ))
 
+    from ..optimization import cull
```

Comment:
Trying to import this outside the function results in an `ImportError` during testing. So imported it locally only where it is needed to avoid this issue.